scripts/BUILD.bazel

```diff
@@ -70,3 +77,23 @@ kt_jvm_binary(
         "//scripts/src/java/org/oppia/android/scripts/regex:regex_pattern_validation_check_lib",
     ],
 )
+
+kt_jvm_binary(
+    name = "proto_check",
```

Comment:
There doesn't seem to be any source corresponding to this binary in the PR.

Reply:
It was unwanted, therefore removed.

Reply:
Verfied.